doc/source/reference/c-api.config.rst

```diff
@@ -101,3 +101,21 @@ Platform information
     Returns the endianness of the current platform.
     One of :c:data:`NPY_CPU_BIG`, :c:data:`NPY_CPU_LITTLE`,
     or :c:data:`NPY_CPU_UNKNOWN_ENDIAN`.
+
+
+Compiler directives
+-------------------
+
+.. c:var:: NPY_LIKELY
+.. c:var:: NPY_UNLIKELY
+
+
+Interrupt Handling
+------------------
+
+.. c:var:: NPY_INTERRUPT_H
+.. c:var:: NPY_SIGSETJMP
+.. c:var:: NPY_SIGLONGJMP
+.. c:var:: NPY_SIGJMP_BUF
+.. c:var:: NPY_SIGINT_ON
+.. c:var:: NPY_SIGINT_OFF
```

Comment:
I think `NPY_UNUSED` can be with the "Compiler Directives", it is used to prevent a compiler warning when the function parameter is unused.  The issue mentions `NPY_TYPES`, but it is already [documented](http://www.numpy.org/devdocs/reference/c-api.dtype.html#c.NPY_TYPES).